legrpc/resolver.go

```diff
@@ -54,13 +54,16 @@ func (r *Resolver) handleEntry(rentry *entry.RaceEntry) {
 
 	// If the current term hasn't expired yet, set the
 	// hostport appropriately
-	state.Addresses = []resolver.Address{resolver.Address{
-		Addr: rentry.HostPort,
-		// this field intentionally left blank (per advice in
-		// the library's docstring)
-		ServerName: "",
-		Attributes: nil,
-	}}
+	for _, hostPort := range rentry.HostPort {
+		state.Addresses = append(state.Addresses, resolver.Address{
```

Comment:
We know exactly how many `Addresses` we'll be adding, so let's construct it with the right size and assign to the right index directly so it's clear that there's a `1:1` relationship. (and for a bit of efficiency)